core/src/collections.rs

```diff
@@ -7,32 +7,14 @@ use crate::{
         BsonTypeName,
     },
     stmt::MongoStatement,
-    util::to_name_regex,
+    util::{add_table_type_filter, to_name_regex, COLLECTION, TABLE, TIMESERIES},
     Error,
 };
-use bson::{doc, Bson, Document};
-use constants::SQL_ALL_TABLE_TYPES;
+use bson::Bson;
 use lazy_static::lazy_static;
 use mongodb::results::CollectionSpecification;
 use mongodb::sync::Cursor;
 use odbc_sys::Nullability;
-use regex::{RegexSet, RegexSetBuilder};
```

Comment:
I moved this stuff into util since we need it in two places now. Perhaps should move the stuff from query I am using to util also.